NodeJS/src/models/AddressBookTypes.ts

```diff
@@ -29,3 +29,42 @@ export interface AddressBookSearchResponse {
   numberOfElements: number
   size: number
 }
+
+export function buildQueryParams(request?: AddressBookSearchRequest) {
```

Comment:
Could you make this function a method of AddressBookSearchRequest?  Then you do not have pass the request object around.  And you have access to all the fields of via 'this'.

Reply:
ok, I'll try to figure that out.

Reply:
@yangAtSpoken could you give some examples? I cannot figure that out.
TypeScript interface cannot have a method with implementation, and if I convert AddressBookSearchRequest to a class I'm getting deserialization errors in the tests about missing value for buildQueryParams() method (which can't be optional).